### ISSUE TITLE
No web interface when Bluetooth Tracker is enabled 0.84.2

### ISSUE THREAD

**Issue by TeoR4eg** [home-assistant/hassio#858]
Home Assistant release 0.84.2
Runing on hassbian

If device tracker [Bluetooth Tracker](https://www.home-assistant.io/components/device_tracker.bluetooth_tracker/) is used, home assistant web interface does not start
Home assistant starts without any (critical) errors : 

```
● home-assistant@homeassistant.service - Home Assistant for homeassistant
   Loaded: loaded (/etc/systemd/system/home-assistant@homeassistant.service; enabled; vendor preset: enabled)
   Active: active (running) since Sun 2018-12-16 09:12:40 MSK; 2min 54s ago
 Main PID: 8609 (hass)
   CGroup: /system.slice/system-home\x2dassistant.slice/home-assistant@homeassistant.service
           ├─8609 /srv/homeassistant/bin/python3 /srv/homeassistant/bin/hass
           ├─8705 /bin/sh -c gatttool --device=4c:65:a8:d4:9e:14 --char-write-req -a 0x10 -n 0100 --adapter=hci0 --listen
           └─8706 gatttool --device=4c:65:a8:d4:9e:14 --char-write-req -a 0x10 -n 0100 --adapter=hci0 --listen

Dec 16 09:14:03 hassbian hass[8609]: 2018-12-16 09:14:03 WARNING (MainThread) [homeassistant.components.sensor] Updating mitemp_bt sensor took longer than the scheduled update interval 0:00:30
Dec 16 09:14:08 hassbian hass[8609]: 2018-12-16 09:14:08 ERROR (Thread-14) [miio.device] Unable to discover a device at address 192.168.1.179
Dec 16 09:14:08 hassbian hass[8609]: 2018-12-16 09:14:08 ERROR (MainThread) [homeassistant.components.light.xiaomi_miio] Got exception while fetching the state: Unable to discover the device 192.168.1.179
Dec 16 09:14:34 hassbian hass[8609]: 2018-12-16 09:14:34 WARNING (MainThread) [homeassistant.components.sensor] Updating mitemp_bt sensor took longer than the scheduled update interval 0:00:30
Dec 16 09:14:38 hassbian hass[8609]: 2018-12-16 09:14:38 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor._humidity is taking over 10 seconds
Dec 16 09:14:39 hassbian hass[8609]: 2018-12-16 09:14:39 ERROR (Thread-20) [miio.device] Unable to discover a device at address 192.168.1.179
Dec 16 09:14:39 hassbian hass[8609]: 2018-12-16 09:14:39 ERROR (MainThread) [homeassistant.components.light.xiaomi_miio] Got exception while fetching the state: Unable to discover the device 192.168.1.179
Dec 16 09:15:05 hassbian hass[8609]: 2018-12-16 09:15:05 WARNING (MainThread) [homeassistant.components.sensor] Updating mitemp_bt sensor took longer than the scheduled update interval 0:00:30
Dec 16 09:15:10 hassbian hass[8609]: 2018-12-16 09:15:10 ERROR (Thread-12) [miio.device] Unable to discover a device at address 192.168.1.179
Dec 16 09:15:10 hassbian hass[8609]: 2018-12-16 09:15:10 ERROR (MainThread) [homeassistant.components.light.xiaomi_miio] Got exception while fetching the state: Unable to discover the device 192.168.1.179

```

Running `sudo netstat --tcp --listening --programs --numeric`
Shows that nothing is using 8123 port:  

```
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 127.0.0.1:4711          0.0.0.0:*               LISTEN      677/pihole-FTL
tcp        0      0 0.0.0.0:9001            0.0.0.0:*               LISTEN      454/mosquitto
tcp        0      0 0.0.0.0:139             0.0.0.0:*               LISTEN      654/smbd
tcp        0      0 0.0.0.0:80              0.0.0.0:*               LISTEN      635/lighttpd
tcp        0      0 0.0.0.0:53              0.0.0.0:*               LISTEN      677/pihole-FTL
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      615/sshd
tcp        0      0 0.0.0.0:1883            0.0.0.0:*               LISTEN      454/mosquitto
tcp        0      0 0.0.0.0:445             0.0.0.0:*               LISTEN      654/smbd
tcp6       0      0 ::1:4711                :::*                    LISTEN      677/pihole-FTL
tcp6       0      0 :::139                  :::*                    LISTEN      654/smbd
tcp6       0      0 :::80                   :::*                    LISTEN      635/lighttpd
tcp6       0      0 :::53                   :::*                    LISTEN      677/pihole-FTL
tcp6       0      0 :::22                   :::*                    LISTEN      615/sshd
tcp6       0      0 :::1883                 :::*                    LISTEN      454/mosquitto
tcp6       0      0 :::445                  :::*                    LISTEN      654/smbd
```

Disabling bluetooth tracker completelly fixes that : 

```
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 127.0.0.1:4711          0.0.0.0:*               LISTEN      677/pihole-FTL
tcp        0      0 0.0.0.0:9001            0.0.0.0:*               LISTEN      454/mosquitto
tcp        0      0 0.0.0.0:139             0.0.0.0:*               LISTEN      654/smbd
tcp        0      0 0.0.0.0:80              0.0.0.0:*               LISTEN      635/lighttpd
tcp        0      0 0.0.0.0:53              0.0.0.0:*               LISTEN      677/pihole-FTL
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      615/sshd
tcp        0      0 0.0.0.0:8123            0.0.0.0:*               LISTEN      8852/python3
tcp        0      0 0.0.0.0:1883            0.0.0.0:*               LISTEN      454/mosquitto
tcp        0      0 0.0.0.0:445             0.0.0.0:*               LISTEN      654/smbd
tcp6       0      0 ::1:4711                :::*                    LISTEN      677/pihole-FTL
tcp6       0      0 :::139                  :::*                    LISTEN      654/smbd
tcp6       0      0 :::80                   :::*                    LISTEN      635/lighttpd
tcp6       0      0 :::53                   :::*                    LISTEN      677/pihole-FTL
tcp6       0      0 :::22                   :::*                    LISTEN      615/sshd
tcp6       0      0 :::1883                 :::*                    LISTEN      454/mosquitto
tcp6       0      0 :::445                  :::*                    LISTEN      654/smbd

```
Reverting back to 0.83 without changing any configs fixes it too

**Comment by TeoR4eg** [comment]
Seems to be fixed in 0.84.6

**Comment by sergeolkhovik** [comment]
Unfortunately have to say that I got this problem, running 0.92.2.
But this is haspbain, not sure if I should write here and create another issue?
Anyway my steps were:
- I have 3 tracking components initialized: owntracks (via UI -> Integrations) and mikrotik + bluetooth in configuration file
- I got owntracks and mikrotik working, but not bluetooth
- after sometime I found that HA code claims that there's no libbluetooth.so.3 available, so I just installed libbluetooth3 package
- after HA restart I got 2 phones added into known_devices file
- I decided to add these 2 devices into persons records, and restarted HA
- after that I got now web UI running
- I commented those 2 lines I added into persons - no luck
- then I found that I have to comment out bluetooth_tracker in device_tracker section configuration, only when I disable BT I get web UI running

I don't know what's the reason of that, how can I debug this better?
